libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -1395,6 +1395,69 @@ describe('File attachment', () => {
     await expectAsync(fixture.nativeElement).toBeAccessible();
   });
 
+  it('should not render form errors when label text is not set', async () => {
```

Comment:
There's no `await` in this function, so there's no need to make the test function async. The same goes for other instances in this file.